services/horizon/internal/actions_metrics.go

```diff
@@ -39,6 +39,7 @@ func (action *MetricsAction) LoadSnapshot() {
 		case metrics.Counter:
 			values["count"] = metric.Count()
```

Comment:
we can't identify that `Counter` metric is of this type since `"count"` key is present in other metric types as well (see below this line). also it's incosistent because `"type"` is found in some cases in this switch clause but is missing in others. add `"type"` to all cases, no cherrypicking.